toolkits/landcover/test/unit/landsat8.unit.test.js

```diff
@@ -18,3 +18,52 @@
 var assert = require('assert');
 var lct = require('../../api');
 
+withEarthEngineStub('Landsat8', function() {
+  it('fmaskCloudsAndShadows()', function() {
+    var dataset = lct.Landsat8();
+
+    // Create spy to capture to invocation of updateMask().
+    spyOn(ee.Image.prototype, 'updateMask').and.callThrough();
+
+    dataset = dataset.fmaskCloudsAndShadows();
+
+    // Verify updateMask() was called with the appropriate mask.
+    expect(ee.Image.prototype.updateMask)
+        .toHaveBeenCalledWith(eeObjectMatching({
```

Comment:
I dont understand; shouldn't this be your "human readable" format?

Reply:
This is the closest to a human readable graph I could get. Any suggestions welcome.

Reply:
I also considered compressing this into a string expression. For example, something like:
```js
Image.and({
  image1: Image.bitwiseAnd({
    image1: Image.select({
      input: uninitializedVar,
      bandSelectors: ['pixel_qa']
    },
    image2: Image.constant({
      value: 8
    }))
    // etc...
  })
})
```

I didn't do this because this representation will look very similar to actual EE API calls, which I didn't want the reader to confuse with actual code they can copy and paste. Any thoughts?